{"uuid": "f94ad2c6-4afd-4683-b025-2e02c36cb925", "children": ["f8505e17-fa69-4db7-b487-834f946917e9"], "befores": [{"name": "create_section", "status": "broken", "statusDetails": {"message": "KeyError: 'id'\n", "trace": "  File \"/home/berserker/python/api_automation_2024/venv/lib/python3.12/site-packages/pluggy/_callers.py\", line 166, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/berserker/python/api_automation_2024/venv/lib/python3.12/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"/home/berserker/python/api_automation_2024/venv/lib/python3.12/site-packages/pluggy/_callers.py\", line 102, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/berserker/python/api_automation_2024/venv/lib/python3.12/site-packages/_pytest/fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/berserker/python/api_automation_2024/venv/lib/python3.12/site-packages/_pytest/fixtures.py\", line 910, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"/home/berserker/python/api_automation_2024/todo_api/conftest.py\", line 48, in create_section\n    id_section_created = section_created[\"body\"][\"id\"]\n                         ~~~~~~~~~~~~~~~~~~~~~~~^^^^^^\n"}, "start": 1712608492800, "stop": 1712608493130}], "start": 1712608492800, "stop": 1712608493149}